test/util/ChartUtils.getLegendProps.spec.tsx

```diff
@@ -0,0 +1,494 @@
+import React, { ReactNode } from 'react';
```

Comment:
a little confused at the file naming convention here. You're just doing this because `ChartUtils` is a giant file?

Tbh I might prefer a folder approach `ChartUtils/getLegendProps.spec.tsx`

Reply:
This is because I added a mock `jest.mock('../../src/util/ReactUtils');` and that broke the other tests. I would prefer to have functions and utils not read things from a DOM but that's going to take some time until we get there.

Also yes it's a giant file with giant tests.

That said, it should work the same in a folder, I can change that.

Reply:
Lol yep you and me both. 

Thanks!

Reply:
The alternative is to just take getLegendProps out into a separate file.